Header header
uint64 framenumber
geometry_msgs/Pose2D[] points
